doc/instanceVisualization.md

```diff
@@ -22,13 +22,21 @@ To create this new App Registration:
   in the Azure Portal
 - click on the _“+ New registration”_ button
   - choose the name of your Function App (you can use the pattern
-    `<your_solution_name><staging_env>ScenarioDownloadFunctionApp` (e.g. _“BreweryDevScenarioDownloadFunctionApp”_)
+    `<your_solution_name><staging_env>ScenarioDownloadFunctionApp`, e.g. _“BreweryDevScenarioDownloadFunctionApp”_)
```

Comment:
maybe missing ( symbol ?

Reply:
Actually, no, the point of this change is to remove an extra open parenthesis :slightly_smiling_face: